services/core-api/app/api/parties/party_appt/models/mine_party_appt.py

```diff
@@ -149,6 +149,7 @@ def json(self, relationships=[]):
             str(self.start_date) if self.start_date else None,
             'end_date':
             str(self.end_date) if self.end_date else None,
+            'is_draft': str(self.is_draft),
```

Comment:
out of scope, but the mine party appointment endpoint doesn't marshal with the response model, it sends back the json. Also there's 2 response models. 🤷‍♀️